src/IJulia/setup.jl

```diff
@@ -26,12 +27,12 @@ import IJulia
 import IJulia: metadata, display_dict
 using  IJulia.CommManager
 import IJulia.CommManager: register_comm
-import Base: writemime, mimewritable
+import Base: show, mimewritable
 
 const comms = Dict{Signal, Comm}()
 
 function get_data_dict(value, mimetypes)
-    dict = Dict{ASCIIString, ByteString}()
+    dict = Dict{Compat.String, Compat.String}()
```

Comment:
To avoid breaking IJulia on 0.4, perhaps this should be `{Compat.ASCIIString, Compat.String}`